.github/workflows/memory-leak-testing.yml

```
@@ -16,6 +16,24 @@ jobs:
         with:
           node-version: ${{ vars.NODE_VERSION }}
 
+<<<<<<< HEAD
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 19: could not find expected ':'

(syntax-check)